js-user-registration-service/src/test/java/com/jumia/warmup/jsuserregistrationservice/rest/UserRegistrationRestIntegrationTest.java

```diff
@@ -0,0 +1,75 @@
+package com.jumia.warmup.jsuserregistrationservice.rest;
+
+import static org.junit.Assert.assertEquals;
+
+import com.jumia.warmup.jsuserregistrationservice.dtos.AccountInformationDTO;
+import com.jumia.warmup.jsuserregistrationservice.dtos.ContactsDTO;
+import com.jumia.warmup.jsuserregistrationservice.dtos.PersonalDetailsDTO;
+import com.jumia.warmup.jsuserregistrationservice.dtos.UserDTO;
+import com.jumia.warmup.jsuserregistrationservice.repositories.UserRepository;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class UserRegistrationRestIntegrationTest {
+
+    public static final String API_USERS = "/api/users";
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    UserDTO userDTO;
+
+    @Before
+    public void setUp() {
+
+        this.userDTO = new UserDTO(new AccountInformationDTO("saraTest", "aaZZa44@")
+            , new ContactsDTO("01234774774", "saraSalah@gmail.com")
+            , new PersonalDetailsDTO("Sara", "Salah", 20));
+    }
+
+    @Test
+    public void registerUser_1_created() {
```

Comment:
For future reference tests can be renamed to follow our test naming convention which is:
methodName_whenSomething_returnsSomething OR methodName_withSomething_thenSomething.

So in this case can be something like:  `registerUser_withValidPayload_userCreated()`

Reply:
Done.